src/hooks/useOktaAxios.js

```diff
@@ -8,7 +8,7 @@ export default function useOktaAxios() {
   return axios.create({
     baseURL: process.env.REACT_APP_BACKENDURL || '',
     headers: {
-      authorization: isAuthenticated ? `${idToken.value}` : '',
+      authorization: isAuthenticated ? `Bearer ${idToken.value}` : '',
```

Comment:
A simple change, but a valuable and insightful one in light of the necessary changes shown in the [paired backend changes](https://github.com/Lambda-School-Labs/human-rights-first-police-be-a/pull/90). By including this authentication we grow closer to a working, secure application - great work!